accessing-betydb-with-arcmap.md

```diff
@@ -0,0 +1,174 @@
+# Accessing BetyDB from ArcMap
+Interested researchers can access BetyDB directly from ArcMap. In some
+cases direct access can simplify manipulation of the BetyDB data but
+this convenience must be weighed against a more complex setup, limits of
+ArcMap compatibility, and additional complexity of extracting data from
+a SQL database.
+
+Accessing the production BetyDB requires creating a secure shell tunnel
+(SSH) to a remote server. After creating the tunnel the database is accessed
+as though it was available on the local machine. A step-by-step process
+is given below.
+
+### Request Access
+Start by [requesting access to the
+BetyDB](https://identity.ncsa.illinois.edu/join/TU49BUUEDM)
+(https://identity.ncsa.illinois.edu/join/TU49BUUEDM) by following the
+link. This will take you to the NCSA identity service. If you do not have
+an NCSA account, you will be asked to create one. This account and
+password will be used to login to the database server. 
+
+Access will generally be granted within 24-hours.
```

Comment:
This approach is supported by NCSA and designed to be public. Using the link will not allow users to self-register. They are only requesting access that will be approved manually by Rob.